doc/source/reference/io.rst

```diff
@@ -25,6 +25,19 @@ Flat file
    DataFrame.to_csv
    read_fwf
 
+.. currentmodule:: pandas.io.parsers
+
+.. autosummary::
+   :toctree: api/
+
+   TextFileReader
+
+   TextFileReader.get_chunk
+   TextFileReader.close
+   TextFileReader.read
```

Comment:
`close` and `read` can probably be public but more important are the magic methods `__enter__`, `__exit__`, and `__next__`. Ideally, people interact with `TextFileReader` in this manner:

```py
with pd.read_csv("test.csv", iterator=True) as reader:
    for chunk in reader:
        print(chunk)
```

Reply:
Would need to add a doc-string to `read` (and maybe also to `close`)